tests/basics/generator1.py

```diff
@@ -20,3 +20,9 @@ def f(x):
 #print(gen.__next__())
 #print(gen.__next__())
 #print(gen.__next__())
+
+try:
+    import uio as io
+except ImportError:
+    import io
+print(f, file=io.StringIO())
```

Comment:
Actually, a simpler way to test the printing is to do `repr(f)`.  Since the text will differ between uPy and CPy, just print the first character of the string, ie `print(repr(f)[0])`.